tests/Workflow/MoveJatsAssetsTest.php

```diff
@@ -561,4 +643,40 @@ public function it_uses_the_extension_if_an_octet_stream_content_type_is_returne
 
         $this->assertXmlStringEqualsXmlString($expected, $task->getDocument());
     }
+
+    /**
+     * @test
+     */
+    public function it_fails_if_the_content_type_is_unknown() : void
+    {
+        $mover = new MoveJatsAssets('.+', 'http://public-assets/path', $this->filesystem, $this->client);
+
+        $document = FluentDOM::load(
+            <<<XML
+<item xmlns="http://libero.pub">
+    <article xmlns="http://jats.nlm.nih.gov" xmlns:xlink="http://www.w3.org/1999/xlink">
+        <body>
+            <graphic xlink:href="http://origin-assets/assets/figure.foo"/>
+        </body>
+    </article>
+</item>
+XML
+        );
+
+        $marking = new Marking();
+        $task = new PutTask('service', ItemId::fromString('id'), ItemVersionNumber::fromInt(1), $document);
+        $transition = new Transition('transition', 'place1', 'place2');
+
+        $event = new Event($task, $marking, $transition);
+
+        $this->mock->save(
+            new Request('GET', 'http://origin-assets/assets/figure.foo'),
+            new Response(200, [], 'figure')
```

Comment:
can we make it a bit more clear that this is the `Response` that is at fault for not having a `Content-Type`? It gets lost in the rest of the setup but it's what makes this test different from the others